src/vs/code/electron-browser/sharedProcess/sharedProcessMain.ts

```diff
@@ -197,7 +197,7 @@ class SharedProcessMain extends Disposable {
 		// Lifecycle
 
 		// Product
-		const productService = { _serviceBrand: undefined, ...product };
+		const productService = { $serviceBrand: undefined, ...product };
```

Comment:
Open to alternate name suggestions for this too. I know we use `$` for ipc methods today